IdentityCore/src/broker_operation/response/MSIDBrokerNativeAppOperationResponse.m

```diff
@@ -117,12 +126,33 @@ - (NSDictionary *)jsonDictionary
     json[MSID_BROKER_OPERATION_RESULT_JSON_KEY] = [@(self.success) stringValue];
     json[MSID_BROKER_OPERATION_RESPONSE_TYPE_JSON_KEY] = self.class.responseType;
     json[MSID_BROKER_APP_VERSION_JSON_KEY] = self.clientAppVersion;
+    json[MSID_BROKER_RESPONSE_GENERATION_TIMESTAMP] = [self.responseGenerationTimeStamp msidDateToFractionalTimestamp:10];
+    json[MSID_BROKER_REQUEST_RECEIVED_TIMESTAMP] = [self.requestReceivedTimeStamp msidDateToFractionalTimestamp:10];
```

Comment:
If self.requestGenerationTimestamp is nil, I think this will return 0.0, Is that expected?

Reply:
Yes, that is okay. I've added unit tests to make sure result is expected if those are missing.